cli-gui-integration.Rmd

```diff
@@ -14,10 +16,72 @@ of flexibility and power waiting for you with the command line. Many times
 you can do something on the command line in a fraction of the time you
 can do it with a GUI.
 
+- that's right; even more important you can easily automate repetitive tasks
+- and it is much more reproducible (what science ought to be). Think also of your future you, in a couple of months from now you really have no idea any longer how exactly you have done complex geoprocessing consisting of several steps.
+- if you are missing a spatial function just write it yourself
+
 But there are times when it's useful to 'point and click'.
 
-<!-- Idea: bullet points of times when GUI-based systems (particularly QGIS) are advantageous. -->
+<!-- Idea: bullet points of times when GUI-based systems (particularly QGIS) are advantageous.
+- digitizing (though there is a new package mapedit from Tim Appelhans that allows for some interactive digitizing in R)
+- georeferencing
+- stereoscopic mapping (think of LiDAR and structure from motion stuff)
+- map production (in case you only want to create a beautiful map once. If you have to produce it over and over again, then maybe CLI is the better approach)
+- zooming and dragging on WMS (though this is also possible with mapview and leaflet)
+- -->
 
-This chapter will outline how you can use portable file formats, packages such as **RQGIS**, **RSAGA** and **rgrass7**, alongside good knowledge of the strengths and weaknesses of 'CLIs vs GUIs'.
+## (R)QGIS
```

Comment:
What you have done in this section is fantastic - thank you, look forward to testing. Very happy with this as-is.